index.html

```diff
@@ -0,0 +1,195 @@
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
```

Comment:
As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51), kindly use a display: flex property for the container of each listed project.

Here's it's style currently:
![image](https://user-images.githubusercontent.com/20554583/205469579-ec50ead2-f2ee-49ef-8115-01c838db18ad.png)

Reply:
hey bro, i used flex there I can show you a screenshot, there's no way I could have positioned that thing without that outside of using position relative which i didn't use

Reply:
Hi @stino-x

Thanks for your response. However, you did not seem to have a style defined for the project-items class. Hence, the use of the default style for a div element. Kindly confirm this.

Reply:
https://drive.google.com/file/d/1O8oCHWOWtt8QC9TVqRT41kusbAecbTK5/view?usp=sharing

that is a screen-shot of my css via my drive

Reply:
> Hi @stino-x
> 
> Thanks for your response. You did not seem to have a style defined for the project-items class., hence, the use of the default style for a div element. Kindly confirm this.

i could do a video as well if you want

Reply:
> Hi @stino-x
> 
> Thanks for your response. You did not seem to have a style defined for the project-items class., hence, the use of the default style for a div element. Kindly confirm this.

a video showing what happens when i take it out:
https://drive.google.com/file/d/1k7UOi_vGn21cpk-qb3MjV8fKgh46fYiF/view?usp=sharing

Reply:
Hi @stino-x,

Thank you for the illustrations. I can see you have put in a lot of work in the project and deserves to be rewarded for that, therefore I can see your project has been approved.

To be clearer, `H-R-J` class as you have it styles the `ul` element containing the technologies listing. However, the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51) stated that the individual cards of the works section be laid out using css flexbox. This means that the container having all the content of a particular project card which in your case is `project-items` should have a `display: flex` property.

I hope this helps to clarify. Once again, congratulations and happy coding👏👏👏.